docker-compose.yml

```diff
@@ -55,7 +64,7 @@ services:
       POSTGRES_PASSWORD: ${POSTGRES_PASSWORD}
       POSTGRES_DB: ${POSTGRES_NAME}
     healthcheck:
-      test: ["CMD-SHELL", "PGPASSWORD=$POSTGRES_PASSWORD pg_isready -U $POSTGRES_USER -d $POSTGRES_DB -p 5432"]
+      test: ["CMD-SHELL", "PGPASSWORD=$POSTGRES_PASSWORD pg_isready -U $POSTGRES_USERNAME -d $POSTGRES_NAME -p 5432"]
```

Comment:
Using `POSTGRES_USERNAME` and `POSTGRES_NAME` won't work as command is executed inside running container, and in the `environment` above we are overwriting the name `POSTGRES_USERNAME->POSTGRES_USER` and `POSTGRES_NAME->POSTGRES_DB` .

Also, once fixed, the command will work on mac/linux, but fail on windows. That's why I did it different way in [this PR](https://github.com/opendexcom/formul.ai/pull/114/files#diff-e45e45baeda1c1e73482975a664062aa56f20c03dd9d64a827aba57775bed0d3R64)

Reply:
nope i checked that it is working now becouse the helthcheck is not considering the enviromet set to service but using the .env file in project